kiliautoml/models/_hugging_face_named_entity_recognition_model.py

```diff
@@ -459,6 +492,40 @@ def _predicted_tokens_to_kili_annotations(
             text, pp_labels, predicted_probas, tokens, null_category, offset_in_text
         )
 
+    def evaluation(self, trainer):
+        train_metrics = trainer.evaluate(trainer.train_dataset)
+        val_metrics = trainer.evaluate()
+        model_evaluation = {}
+        nb_train_ent = 0
+        nb_val_ent = 0
+        for label in categories_from_job(self.job):
```

Comment:
Those kinds of function should be a method in the class Job

Reply:
ticket to create a Class Job